README.md

```diff
@@ -108,6 +106,44 @@ True
 False
 ```
 
+### Traversing (inspecting trie internals)
+
+```python
+>>> from trie import HexaryTrie
+>>> t = HexaryTrie(db={})
+>>> t.root_hash
+b'V\xe8\x1f\x17\x1b\xccU\xa6\xff\x83E\xe6\x92\xc0\xf8n[H\xe0\x1b\x99l\xad\xc0\x01b/\xb5\xe3c\xb4!'
+>>> t[b'my-key'] = b'some-value'
+>>> t[b'my-other-key']  = b'another-value'
+
+# Look at the root node:
+>>> root_node = t.traverse(())
+>>> root_node
+HexaryTrieNode(sub_segments=((0x6, 0xd, 0x7, 0x9, 0x2, 0xd, 0x6),), value=b'', suffix=(), raw=[b'\x16\xd7\x92\xd6', b'\xb4q\xb8h\xec\x1c\xe1\xf4\\\x88\xda\xb4\xc1\xc2n\xbaw\xd0\x9c\xf1\xacV\xb4Dk\xa7\xe6\xd7qf\xc2\x82'])
```

Comment:
Thanks for recommending the docs. Here it is. Hopefully it gives more of a feel for how the traversal API works in practice.

(Not specifically seeking feedback, just thought you might be interested)